src/solve.c

```diff
@@ -99,7 +101,7 @@ void set_run_sequence(void)
 			break;
 		run_sequence[i++] = 'F';
 	}
-	run_sequence[0] = 'B';
+	run_sequence[i++] = 'F';
```

Comment:
The `run_sequence[0] = 'B';` was moved up a bit to line 80 (`run_sequence[i++] = 'B';`).

The current raw path generation uses `S` (start) and `B` (stop) movements as not-really-movements, but rather "states". That means:

- To start we now need `SF` (front out of the starting cell) instead of `S` as before
- To stop we now need `FB` (front into the cell and stop) instead of `B` as before

Reply:
Ok